crates/apollo-compiler/src/database/hir.rs

```diff
@@ -1205,11 +1315,16 @@ impl Field {
         }
     }
 
-    /// Get a reference to the field's name.
+    /// Get the field's name, corresponding to the definition it looks up.
     pub fn name(&self) -> &str {
         self.name.src()
     }
 
+    /// Get the field's response name.
+    pub fn response_name(&self) -> &str {
```

Comment:
i don't follow what `response_name` means here? :(

Reply:
ok ok that makes sense. can you add an explanation to the doc comment, in case someone else is also confused?